src/locales/en.json

```
@@ -29,5 +29,11 @@
     "share": "Share with",
     "report": "Report",
     "copiedSuccessfully": "Link copied successfully!"
+  },
+  "search": {
+    "recentsSearch": "Recent search",
```

Would also need to be updated on other places
```suggestion
    "recentSearches": "Recent searches",
```